packages/graphql-language-service-config/package.json

```diff
@@ -17,7 +17,7 @@
   "keywords": [
     "graphql"
   ],
-  "main": "dist/index.js",
+  "main": "src/index.js",
```

Comment:
Thanks @RomanGotsiy! Could you actually un-do all these `package.json` changes except for the one on the root dir?